src/editor/components/styles/Image.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../../styles/slugline_vars";
+
+.Image {
+  width: 100%;
+  // match the spacing for paragraphs
+  margin-bottom: 1rem;
+
+  &_img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+
+    max-width: 85%;
+    min-width: 50%;
+
+    height: auto;
+
+    &--selected {
+      box-shadow: 0px 0px 0px 2px lightblue;
```

Comment:
Is this a common selection style? If so, would this be better expressed as a mixin or pseudo-selector?

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/84/commits/a257e0d1fafbea9d55ec1c8df5f29c3313bdbcca